sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1003,14 +1001,15 @@ export class ApplicationService extends RecordDataModelService<Application> {
     application.applicationStatusUpdatedOn = now;
     application.modifier = { id: auditUserId } as User;
     application.updatedAt = now;
+
+    // Updates the current assessment status to cancellation requested.
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.CancellationRequested;
+    application.currentAssessment.modifier = { id: auditUserId } as User;
```

Comment:
When used multiple times we can create a variable to reuse the `{ id: auditUserId } as User`.
In this case in lines 1002 and 1008.